pkg/delete/delete.go

```diff
@@ -46,22 +49,35 @@ func (d DeletePointer) asCoordinate() coordinate.Coordinate {
 	}
 }
 
+type ClientSet struct {
+	DTClient         dtclient.Client
+	AutomationClient *automationClient.Client
+}
+
 // Configs removes all given entriesToDelete from the Dynatrace environment the given client connects to
-func Configs(client dtclient.Client, apis api.APIs, entriesToDelete map[string][]DeletePointer) []error {
+func Configs(clients ClientSet, apis api.APIs, automationResources map[string]config.AutomationResource, entriesToDelete map[string][]DeletePointer) []error {
 	errs := make([]error, 0)
 
-	for targetApi, entries := range entriesToDelete {
-		theApi, found := apis[targetApi]
-
-		// handle settings 2.0 objects
-		if !found {
-			deleteErrs := deleteSettingsObject(client, entries)
+	for entryType, entries := range entriesToDelete {
+		if targetApi, isApi := apis[entryType]; isApi {
+			deleteErrs := deleteClassicConfig(clients.DTClient, targetApi, entries, entryType)
 			errs = append(errs, deleteErrs...)
+		} else if targetAutomation, isAutomation := automationResources[entryType]; isAutomation {
 
-		} else {
-			deleteErrs := deleteClassicConfig(client, theApi, entries, targetApi)
-			errs = append(errs, deleteErrs...)
+			if !featureflags.AutomationResources().Enabled() {
+				continue
+			}
+
+			if clients.AutomationClient == nil {
+				log.Warn("Skipped deletion of %d Automation configurations of type %q as API client was unavailable", len(entries), entryType)
```

Comment:
nit: We currently mix (even in this PR) messages with a trailing dot and without. I'd argue we should use them.
Also, some of the logging starts with a lowercase, also in this PR

Reply:
Addressed the logging for delete in extra commit. Also unified how statements end (`.`) and actions in progress (`...`).

Re casing: I found a single log line that started lowercase and that seems to have been a copy pasted error message. 
Errors start lowercase and don't have punctuation per Go convention.